add document about how to deploy locally (#9333)
NuGet/NuGetGallery: README.md

Refer to [our documentation](./docs/) for information on how to develop the frontend, use AAD, and more.

## Deploy
### Deploy to Azure

You will find instructions on how to deploy the Gallery to Azure [here](https://github.com/NuGet/NuGetGallery/blob/master/docs/Deploying/README.md).

### Deploy locally

> After you succeed in running the NuGet Gallery, you can create a publish to deploy locally(such as your windows computer). 

Reviewer: ```suggestion
After you succeed in running the NuGet Gallery, you can create a Publish configuration to deploy locally (such as your local Windows computer). 
```

> After you succeed in running the NuGet Gallery, you can create a Publish configuration to deploy locally (such as your local Windows computer). 

Reviewer: ```suggestion
After you succeed in running the NuGet Gallery, you can create a publish profile to deploy locally (such as your local Windows computer). 
```

The steps are:
1. Select the `NuGetGallery` project in Solution Explore of Visual Studio. 

> 2. Right click the project, and then click `Publish` in the pop-up menu. Create a publish and make sure the Target is `Folder`

Reviewer: ```suggestion
2. Right click the project, and then click `Publish` in the pop-up menu. Create a Publish configuration and make sure the Target is set to `Folder`.
```

> 2. Right click the project, and then click `Publish` in the pop-up menu. Create a Publish configuration and make sure the Target is set to `Folder`.

Reviewer: ```suggestion
2. Right click the project, and then click `Publish` in the pop-up menu. Create a publish profile and make sure the Target is set to `Folder`.
```

> 3. Copy the contents of the `Target Location` to any folder you want. Assume the folder is `C:\GreenSoftware\NuGetGallery`.

Reviewer: ```suggestion
3. Copy the contents of the `Target Location` to any folder you want. For the following example, assume the folder is `C:\ContosoSoftware\NuGetGallery`.
```

> 4. Run the command below to run it.(The parameter `/path` of iisexpress only supports absolute path on Windows)

Reviewer: ```suggestion
4. Execute the command below to start the web app (note that the parameter `/path` of iisexpress.exe only supports absolute paths on Windows).
```

    ```cmd

>     "C:\Program Files\IIS Express\iisexpress.exe" /path:C:\GreenSoftware\NuGetGallery

Reviewer: ```suggestion
    "C:\Program Files\IIS Express\iisexpress.exe" /path:C:\ContosoSoftware\NuGetGallery
```

    ```

> Now you can access the local website with browsers. The url is `https://localhost`

Reviewer: ```suggestion
Now you can access the local website with a web browser. The URL is `https://localhost`.
```
